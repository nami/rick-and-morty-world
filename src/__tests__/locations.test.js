import React from "react";
import { create } from "react-test-renderer";
import Locations from '../components/locations'

describe("Locations component", () => {
  test("Matches the snapshot", () => {
    const locations = create(<Locations />);
    expect(locations.toJSON()).toMatchSnapshot();
  });
});