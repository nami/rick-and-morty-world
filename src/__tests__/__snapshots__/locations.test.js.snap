// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations component Matches the snapshot 1`] = `
<div>
  <h1
    className="title is-2 has-text-centered"
  >
    Rick & Morty Worlds
  </h1>
  <div
    className="locations"
  />
</div>
`;
